bsp/esp32_p4_function_ev_board/Kconfig

```diff
@@ -75,6 +75,31 @@ menu "Board Support Package(ESP32-P4)"
         help
             LEDC channel is used to generate PWM signal that controls display brightness.
             Set LEDC index that should be used.
+            
+        choice BSP_LCD_COLOR_FORMAT
+            prompt "Select LCD color format"
+            default BSP_LCD_COLOR_FORMAT_RGB565
```

Comment:
Q: Do both displays support both RGB565 and RGB888?

Reply:
yes